lib/fog/aws/models/elb/policy.rb

```diff
@@ -45,12 +44,8 @@ def destroy
           reload
         end
 
-        def reload
-          load_balancer.reload
-        end
-
         def load_balancer
-          collection.load_balancer
+          service.load_balancers.new(identity: collection.load_balancer_id)
```

Comment:
Could you fix the 1.8 incompatible named argument here? Hopefully we'll reach a point where we can drop 1.8 soon, but not quite ready for the major/breaking change I'm afraid. Thanks!